packages/client/src/index.css

```diff
@@ -1,69 +1,22 @@
 :root {
```

Comment:
Could we also remove this css file,
We can set the font via mui theming in another PR